ui/package.json

```diff
@@ -56,5 +56,8 @@
     "devDependencies": {
         "chalk-cli": "^5.0.0",
         "lerna": "^3.22.1"
+    },
+    "dependencies": {
+        "cross-spawn": "7.0.5"
```

Comment:
@sachaudh as mentioned on private channel, is it possible to delete this and still keep change below?

Reply:
For the patch release branch, does that really matter?

Reply:
It attributes this to direct `dependencies` instead of transitive `devDependencies`

Not sure how big an impact on OSSLs/SBOM

Reply:
Looks like I can remove it and `yarn list` returns the same output. So let me do that then